src/zjs_event.c

```diff
@@ -12,6 +12,7 @@
 #define DEFAULT_MAX_LISTENERS   10
 
 static jerry_value_t zjs_event_emitter_prototype = 0;
+zjs_callback_id emit_id = -1;
```

Comment:
making this global doesn't it mean that it can be overwritten with multiple events from different modules?

Reply:
It was already global... I just moved it up from down below because it's lifetime is now tied to that to that of the prototype defined just above.

However, it could probably be static...